MetaMask/metamask-extension
# Changed text for review spending cap copy

## Explanation

Changed text for review spending cap copy instead of Review your spending cap it now shows Review the spending cap for your.

* Fixes #18066

## Screenshot
### Before

![Screenshot 2023-03-17 at 11 41 55](https://user-images.githubusercontent.com/120601632/225882708-8146d252-a79d-4259-afdf-0c8d372cd88e.png)

### After

![Screenshot 2023-03-17 at 11 27 06](https://user-images.githubusercontent.com/120601632/225881838-436dcac2-967d-4ff6-b593-f2b28e422c87.png)

## Unit tests

![Screenshot 2023-03-17 at 11 35 41](https://user-images.githubusercontent.com/120601632/225881807-64c390e5-f8d2-4044-a6c1-eb70e8a71344.png)

## Lint

![Screenshot 2023-03-17 at 11 28 35](https://user-images.githubusercontent.com/120601632/225881824-5b2c19b6-956c-4b56-90fc-9e8e3cbda702.png)

## Manual Testing Steps

1. Login to MetaMask extension
2. Go to https://metamask.github.io/test-dapp/
3. Click Create Token
4. Click Confirm in MetaMask extension and wait transaction to be done
5. Go to https://metamask.github.io/test-dapp/
6. Click Approve Tokens
7.  Put some value in input and click Next
8. Now the screen has title Review the spending cap for your
